packages/samples/src/tools/home.ts

```diff
@@ -5,35 +5,27 @@ import * as agents from "@graphai/agents";
 import { graphDataTestRunner } from "@receptron/test_utils";
 import { home_functions } from "./home_functions";
 
-const home_actions = {
-  fill_bath: { type: "message_template", message: "Success. I started filling the bath tab." },
-  set_temperature: { type: "message_template", message: "Success. I set the temperature to {temperature} for {location}" },
-  start_sprinkler: { type: "message_template", message: "Success. I started the sprinkler for {location}" },
-  take_picture: { type: "message_template", message: "Success. I took a picture of {location}" },
-  play_music: { type: "message_template", message: "Success. I started playing {music} in {location}" },
-  control_light: { type: "message_template", message: "Success. The light switch of {location} is now {switch}." },
-};
```

Comment:
While adding a node might allow this action to be executed like slashAgent, I felt it would be redundant as a sample, so I decided to keep it simple and just output it using stringTemplateAgent.

Reply:
Since this function behaved like calling a dummy function, it is better to use stringTemplateAgent for the demo.
Thank you!